app/models/ad.rb

```diff
@@ -55,6 +56,9 @@ class Ad < ApplicationRecord
   scope :active, -> { not_deleted }
   scope :ordered, -> { order(created_at: :desc) }
   scope :for_kind, ->(kind) { where(kind: kind) }
+  scope :not_reminded, -> { where(reminder_sent_at: nil) }
+  scope :old, -> { where("created_at < ?", 2.days.ago) }
```

Comment:
`old` sounds a bit off.

You could have also written this code below and it would be difficult to know by looking at some code using this scope, without checking the implementation:

```rb
  scope :old, -> { deleted }
```

It's "potentially expired ad". 🤔 
I'm not sure about what would be a better name.
`expiring`, `aged` maybe?

For the scope itself, this should work:
```suggestion
  scope :old, -> { where(created_at: DateTime::Infinity.new(-1)..2.days.ago) }
```

or
```suggestion
  scope :old, -> { where(created_at: ..2.days.ago) }
```

Not sure if it's easier to read.

Reply:
We could also just remove this scope and inline it where it's only being used in `AdReminder` 🤔

Reply:
I like `expiring` and the open range syntax. 👍

Reply:
Switched it to that.